test/resources/config/regexp.bal

```diff
@@ -0,0 +1,24 @@
+function name() {
+
+    string pattern = "ABC";
+    
+    string:RegExp _ = re `abc`;
+    string:RegExp _ = re `(abc)`;
+    string:RegExp _ = re `[abc]`;
+
+    string:RegExp _ = re `\p{gc=Ll}|\P{Lm}|\p{sc=Latin}`;
+    string:RegExp _ = re `\u{12FFFF}`;
+
+    string:RegExp _ = re `${pattern}`;
+    string:RegExp _ = re `\w\s\n`;
+
+    string:RegExp _ = re `(?-:abc)`;
+    string:RegExp _ = re `(?i-:abc)`;
+    string:RegExp _ = re `(?im-sx:abc)`;
+
+    string:RegExp _ = re `(abc)|${pattern}|[abc]|(?i:abc)`;
+    string:RegExp _ = re `[((abc))))\]]`;
+
+    string:RegExp _ = re `[abc\w\p{Ll}\u{FF}]`;
+
```

Comment:
Another possible test case is to use a record/object field access within the interpolation and verify the syntax highlighting.

E.g.:

```
type MyRecord record {
   string myStr = "aaa";
};

MyRecord rA;
```

```
string:RegExp _ = re `${rA.myStr}`;
```

Can apply this for functions defined with objects as well.